### DISCUSSION TITLE
Error JWT

### DISCUSSION THREAD

**Discussion by justformeokay** [supabase/supabase#18421] [Questions]
why my supabase like this, before this problem, i set up email template for magic link
![reportissue](https://github.com/supabase/supabase/assets/146034705/af02b307-1402-483c-8c6d-0c406e35ed66)

**Comment by GaryAustin1** [comment]
I usually see that randomly, occasionally, and have been signed out by the Dashboard.  Refreshing the screen asks to log in again.

**Reply by justformeokay** [reply]
Same as with me.. my local time in my computer is the same with network.. when phone try to get or post request always failed when show error jwt..
